src/routes/write-exercise/WriteExerciseScreen.tsx

```diff
@@ -66,7 +68,7 @@ const WriteExerciseScreen = ({ navigation, route }: WriteExerciseScreenPropsType
 
       <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
         {documents && document && (
-          <ScrollView contentContainerStyle={{ flex: 1 }} keyboardShouldPersistTaps='always'>
+          <KeyboardAwareScrollView contentContainerStyle={{ height }}>
```

Comment:
By setting `flexGrow: 1` i'm getting an issue on android by wrapping the ImageCarousel. Maybe someone finds a nicer solution. On IOS everything works as expected :/

According to @maxammann 's comment on the closed pr. 
I tested:
https://reactnative.dev/docs/keyboardavoidingview in combination with a `ScrollView`
It causes strange scrolling issues each time i open the keyboard, the scroll view increased. So i moved back to this library

Reply:
Just using the ScrollView without a wrapper and setting `flexGrow` seems to work for me on android.

Reply:
its just working for emulator. thats what i experienced